docs/hadoop.adoc

```diff
@@ -4,17 +4,72 @@
 JanusGraph-Hadoop works with TinkerPop's hadoop-gremlin package for
 general-purpose OLAP.
 
-Here's a three step example showing some basic integrated JanusGraph-TinkerPop functionality.
+The current tutorial covers the following topics for setting up Spark environment to work with Janusgraph using Hadoop for doing OLAP.
 
-1. Manually define schema and then load the Grateful Dead graph from a TinkerPop Kryo-serialized binary file 
-2. Run a VertexProgram to compute PageRanks, writing the derived graph to `output/~g`
-3. Read the derived graph vertices and their computed rank values
+1. Bulk loading into backend data store with local spark (Spark dist which comes with JanusGraph)
+2. Bulk loading into backend data store using Spark Cluster in Standalone mode. (The tutorial expects that there is a up and running Spark Cluster as provided host)
+3. Reads from backend data store into Hadoopgraph for OLAP against local spark
+4. Reads from Cassandra backend into Hadoopgraph for OLAP against Spark Cluster in standalone mode.
```

Comment:
To to consistent with 3. Update the title to:
"Reads from backend data store into Hadoopgraph for OLAP against Spark Cluster in standalone mode."